tls/s2n_send.c

```diff
@@ -111,7 +117,7 @@ int s2n_flush(struct s2n_connection *conn, s2n_blocked_status *blocked)
         struct s2n_blob alert = { 0 };
         alert.data = conn->writer_alert_out.blob.data;
         alert.size = 2;
-        POSIX_GUARD(s2n_record_write(conn, TLS_ALERT, &alert));
+        POSIX_GUARD_RESULT(s2n_record_write_all(conn, TLS_ALERT, &alert));
```

Comment:
There are other places where s2n_record_write() is called. Did you want to replace those too?

Reply:
Good point. I was using the existing "bytes written" return value for my unpublished post-handshake fragmentation code, but literally no current call to s2n_record_write correctly uses the return value. I should just check the number of bytes written in the existing s2n_record_write method and figure something else out for post-handshake fragmentation.

Reply:
That change ended up touching a LOT of files (mostly trivially) so I moved it to another PR: https://github.com/aws/s2n-tls/pull/3722